api/v1alpha1/envoyextensionypolicy_types.go

```diff
@@ -46,12 +46,12 @@ type EnvoyExtensionPolicySpec struct {
 	// TargetRef
 	TargetRef gwapiv1a2.PolicyTargetReferenceWithSectionName `json:"targetRef"`
 
-	// WASM is a list of Wasm extensions to be loaded by the Gateway.
+	// Wasm is a list of Wasm extensions to be loaded by the Gateway.
 	// Order matters, as the extensions will be loaded in the order they are
 	// defined in this list.
 	//
 	// +optional
-	WASM []Wasm `json:"wasm,omitempty"`
+	Wasm []Wasm `json:"wasm,omitempty"`
```

Comment:
Struggled between `WASM` and `Wasm`. Looks like `Wasm` is [the official abbreviation](https://webassembly.org/). :-)

Reply:
It looks a bit confused, the name is Wasm and the type is the slice of Wasm ([]Wasm)

Reply:
Yeah, it's not pretty. I'm following the existing ExtProc setting here.

Alternative:

```
Wasms []Wasm `json:"wasm,omitempty"`

ExtProcs []ExtProc `json:"extProc,omitempty"`
```

```
WasmList []Wasm `json:"wasm,omitempty"`

ExtProcList []ExtProc `json:"extProc,omitempty"`
```

@arkodg @guydc

Reply:
Follow existing EG API types to use `Wasms` and `ExtProcs`.

Reply:
We are using acronyms here, let's use Wasm and ExtProc